RELEASE_NOTES.rst

```diff
@@ -14,6 +14,60 @@ ARTIQ-5
   have been renamed to ``ref_time_mu`` for consistency, as they are in machine
   units.
 * :func:`~artiq.tools.verbosity_args` renamed to :func:`~artiq.tools.add_common_args`. New feature: adds an option to print the ARTIQ version.
+* A gateware-level input edge counter has been implemented (driver:
```

Comment:
Normally, release notes contain breaking changes. Those examples should be moved to the manual.